### PR TITLE
t1utils: new recipe

### PR DESCRIPTION
**Pull request by jmairboeck**
Now that we have a guile1 package, I wanted to give porting Lilypond a try. It turned out that this is not so easy (I have suspected that already, as I have already fixed the openSUSE package for Lilypond and am kind-of a co-maintainer for that).

There are a few missing dependencies yet, the biggest of which is Texlive. Another missing tool is t1asm (part of t1utils), which is provided with this pull request.

Tested building on x86_64 and x86_gcc2, it works on both. (yes, using gcc2!)

This is my first HaikuPorts PR, let's see what you think.

### CODE REVIEW

File: media-fonts/t1utils/t1utils-1.41.recipe [hunk 1]
```diff
@@ -0,0 +1,51 @@
+SUMMARY="Utilities for manupulating Adobe Type 1 font software"
+DESCRIPTION="T1utils is a collection of simple programs for manipulating
+	PostScript Type 1 fonts. Together, they allow you to convert between PFA
```
Comment (diversys) [inline media-fonts/t1utils/t1utils-1.41.recipe:3]: You need to wrap new lines with "\" Also no need for indentation here.

File: media-fonts/t1utils/t1utils-1.41.recipe [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	(ASCII) and PFB (binary) formats, disassemble PFA or PFB files into
+	human-readable form, and reassemble them into PFA or PFB format.
+	Additionally you can extract font resources from a Macintosh font file or
+	create a Macintosh Type 1 font file from a PFA or PFB font."
+HOMEPAGE="http://www.lcdf.org/~eddietwo/type/#t1utils"
```
Comment (diversys) [inline media-fonts/t1utils/t1utils-1.41.recipe:8]: We prefer https it possible

File: media-fonts/t1utils/t1utils-1.41.recipe [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+    cmd:awk
```
Comment (diversys) [inline media-fonts/t1utils/t1utils-1.41.recipe:36]: Use tab instead of spaces.

**Comment by jmairboeck** [comment]
I have fixed the issues.

File: media-fonts/t1utils/t1utils-1.41.recipe [hunk 4]
```diff
@@ -0,0 +1,52 @@
+SUMMARY="Utilities for manupulating Adobe Type 1 font software"
+DESCRIPTION="\
+T1utils is a collection of simple programs for manipulating PostScript Type 1 \
```
Comment (diversys) [inline media-fonts/t1utils/t1utils-1.41.recipe:3]: Move the description to the previous line removing leading "\".

**Comment by jmairboeck** [comment]
I just found that the [texlive package](https://github.com/haikuports/haikuports/blob/master/app-text/texlive/texlive-2014.recipe#L321) also provides these utilities and they are also [part of CTAN](https://ctan.org/pkg/t1utils) as it seems. Do we even need this standalone package then?

In openSUSE, t1utils is packaged separately and not part of texlive (currently using version 2017 here).

**Comment by pulkomandy** [comment]
The TeXLive package is too large and unmanageable. So the more we can remove from it and make things more modular, the better.

File: media-fonts/t1utils/t1utils-1.41.recipe [hunk 5]
```diff
@@ -0,0 +1,50 @@
+SUMMARY="Utilities for manupulating Adobe Type 1 font software"
+DESCRIPTION="T1utils is a collection of simple programs for manipulating \
+PostScript Type 1 fonts. Together, they allow you to convert between PFA \
+(ASCII) and PFB (binary) formats, disassemble PFA or PFB files into human-\
```
Comment (Begasus) [inline media-fonts/t1utils/t1utils-1.41.recipe:4]: Maybe start new paragraphs on new lines like:
```diff
-PostScript Type 1 fonts. Together, they allow you to convert between PFA \
-(ASCII) and PFB (binary) formats, disassemble PFA or PFB files into human-\
+PostScript Type 1 fonts.
+Together, they allow you to convert between PFA (ASCII) and PFB (binary) \
formats, disassemble PFA or PFB files into human-readable ...
```

Comment (jmairboeck) [reply media-fonts/t1utils/t1utils-1.41.recipe]: I have copied the description directly from the [README at github](https://github.com/kohler/t1utils/blob/master/README.md#t1utils) but I reformatted it nonetheless.

Comment (Begasus) [reply media-fonts/t1utils/t1utils-1.41.recipe]: That's OK, it just looks nicer in HaikuDepot :)

**Pull request merged**